acadevmy/redis-patterns-console
# upgrade to angular 12

## What does this change?

- updated project to Angular 12
- updated dependencies
- used marked library instead markdown

## Why is this change important?

- Angular
  - performance
  - build optimization
  - fixed issues
- Marked
  - maintained
  - TS definitions
  - markdown.js is abandoned at v0.5.0 😥

## How can I test it?

- check `package.json` dependencies/devDependencies
- run `ng serve`
